configure.ac

```diff
@@ -217,6 +220,10 @@ if test "x$enable_unix_socket" = "xno"; then
     AC_DEFINE([DISABLE_UNIX_SOCKET],1,[Set to nonzero if you want to disable unix domain socket])
 fi
 
+if test "x$enable_linking_jemalloc" = "xyes"; then
+    LIBS="$LIBS -ljemalloc"
```

Comment:
Are there any easy tests we can do here? pkg-config or something?

Reply:
Added `t/jemalloc.t` which uses `ldd` to check the binary.

Reply:
I can't remember what I was thinking at the time, but I bet what I meant was detecting if jemalloc existed on the system before trying to link it. So by "Tests" I meant autoconf tests. That would make more sense, I think? :)

Reply:
Got it. `dpkg` seems to be providing both lib existence and the version.

Reply:
Let me try with `ldconfig`